content/millennium/r4/other/health-cards.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Health Cards | R4 API
+---
+
+# Health-Cards
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+A Health Card is a verifiable, digital artifact that represents a limited set of information about an individual, relevant to a specific health status. The card can be presented in digital and physical form. A QR code representing the card can be shared from a device or a piece of paper. Information in the health card can be used to determine the authenticity of the artifact and its issuer. The goal is to provide individuals with the choice to share limited health information in a portable and secure manner.
+
+Health cards are designed in conformance with the emerging [SMART<sup>®</sup> Health Card framework](https://smarthealth.cards) ([technical specification](https://spec.smarthealth.cards/)) and [HL7<sup>®</sup> FHIR<sup>®</sup> Vaccination and Testing IG](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/).
+
+The [$health-cards-issue](#health-cards-issue) operation adheres to the [via FHIR specification](https://spec.smarthealth.cards/#via-fhir-health-cards-issue-operation) and returns Health Cards as verifiableCredentials in a FHIR [Parameters](http://hl7.org/fhir/r4/parameters.html) resource. These verifiable credentials are [JSON Web Signatures](https://datatracker.ietf.org/doc/html/rfc7515) with a compressed payload. Refer to the specification for details on the protocols for 
```

Comment:
@zsura  Thought I made a comment on this earlier but don't see it. Seems the wording is a little off here "The health-cards-issue operation adheres to the "via FHIR specification".. Should we remove 'via' and have it state, "The health-cards-issue operation adhers to the "FHIR specification and returns...

Reply:
Addressed in https://github.com/cerner/fhir.cerner.com/pull/659/commits/be9ca56ca3eef66b8cf79477ea185ee6a2867f93